src/push.go

```diff
@@ -86,7 +86,18 @@ func PushManyWithGitImpl(ctx context.Context, flags *PushFlags, repoNames []stri
 }
 
 func PushWithGitImpl(ctx context.Context, flags *PushFlags, repoName string, ghClient *github.Client, gitimpl GitImplementation) error {
-	_, nwo, err := extractSourceDest(repoName)
+	org, nwo, err := extractSourceDest(repoName)
```

Comment:
```suggestion
	source, destination, err := extractSourceDest(repoName)
```
this would be more clear

Reply:
Sure but if we change `nwo` to `destination`, there are a bunch of changes to add in the code, not related to my PR and we break the global logical naming rules in the project. Are you sure about it ?